remix.init/package.json

```diff
@@ -3,8 +3,5 @@
   "private": true,
   "main": "index.js",
   "license": "MIT",
-  "dependencies": {
-    "@iarna/toml": "^2.2.5",
-    "sort-package-json": "^1.55.0"
-  }
+  "dependencies": {}
```

Comment:
This file can be removed completely I think, since we don't have any dependencies anymore?

Reply:
Yeah, we can probably remove this 👍

Reply:
Unfortunately, we can't remove until https://github.com/remix-run/remix/pull/3146 is merged/released